saleor/dashboard/product/forms.py

```diff
@@ -14,6 +16,20 @@
 from .widgets import ImagePreviewWidget
 
 
+class RichTextField(forms.CharField):
+    """
+    A field for rich text editor, providing backend sanitization
```

Comment:
`[pydocstyle] D212:Multi-line docstring summary should start at the first line`. Please use `pydocstyle`.